testcase/testcase.go

```diff
@@ -138,6 +188,14 @@ func New(reader io.Reader) (*TestCaseSet, error) {
 			tcs.descriptions = append(tcs.descriptions, tc.Description)
 		}
 	}
+
+	// Convert dot fields
+	if err := tcs.convertDotFileds(); err != nil {
+		return nil, err
+	}
+
+	log.Debugf("%+v", tcs)
```

Comment:
I now check the loglevel before call logger. It permit to keep logger call instead to put and remove it and avoir latency about there.

Reply:
The problem with the logging call isn't the latency but the noise in the log. We can keep it (without a conditional) but it would be useful with a prefix that describe what it is instead of just dumping the map to the log.

Reply:
Ok, i will look that

Reply:
`log.Debugf("Current TestCaseSet after convert fiedls: %+v", tcs)`